package/batocera/core/batocera-configgen/configgen/configgen/generators/daphne/daphneGenerator.py

```diff
@@ -25,19 +25,26 @@ def generate(self, system, rom, playersControllers, gameResolution):
 
         if os.path.isfile(singeFile):
             commandArray = [batoceraFiles.batoceraBins[system.config['emulator']],
-                            "singe", "vldp", "-retropath", "-framefile", frameFile, "-script", singeFile,
-                            "-x", str(gameResolution["width"]), "-y", str(gameResolution["height"]), "-fullscreen",
+                            "singe", "vldp", "-retropath", "-framefile", frameFile, "-script", singeFile, "-fullscreen",
                             "-manymouse", "-datadir", batoceraFiles.daphneDatadir, "-homedir", batoceraFiles.daphneDatadir]
-        elif system.config["ratio"] == "16/9":
-            commandArray = [batoceraFiles.batoceraBins[system.config['emulator']],
-                            romName, "vldp", "-framefile", frameFile, "-useoverlaysb", "2", "-ignore_aspect_ratio",
-                            "-x", str(gameResolution["width"]), "-y", str(gameResolution["height"]), "-fullscreen",
-                            "-fastboot", "-datadir", batoceraFiles.daphneDatadir, "-homedir", batoceraFiles.daphneHomedir]
         else:
             commandArray = [batoceraFiles.batoceraBins[system.config['emulator']],
                             romName, "vldp", "-framefile", frameFile, "-useoverlaysb", "2", "-fullscreen",
                             "-fastboot", "-datadir", batoceraFiles.daphneDatadir, "-homedir", batoceraFiles.daphneHomedir]
 
+        # Aspect ratio
+        if system.config["ratio"] == "4/3":
+            commandArray.append("-force_aspect_ratio")
+        else:
```

Comment:
This 'else' is redundant as video aspect ratio is now implicit.

`-ignore_aspect_ratio` argument is now deprecated.